src/FixtureBuilder/Denormalizer/Fixture/Chainable/NullRangeNameReferenceDenormalizer.php

```diff
@@ -0,0 +1,193 @@
+use Nelmio\Alice\Definition\FlagBag;
+use Nelmio\Alice\Definition\MethodCallBag;
+use Nelmio\Alice\Definition\PropertyBag;
+use Nelmio\Alice\Definition\SpecificationBag;
+use Nelmio\Alice\FixtureBag;
+use Nelmio\Alice\FixtureBuilder\Denormalizer\Fixture\ChainableFixtureDenormalizerInterface;
+use Nelmio\Alice\FixtureBuilder\Denormalizer\Fixture\SpecificationsDenormalizerInterface;
+use Nelmio\Alice\FixtureBuilder\Denormalizer\FlagParserAwareInterface;
+use Nelmio\Alice\FixtureBuilder\Denormalizer\FlagParserInterface;
+use Nelmio\Alice\FixtureInterface;
+use Nelmio\Alice\IsAServiceTrait;
+use Nelmio\Alice\Throwable\Exception\LogicExceptionFactory;
+
+final class NullRangeNameReferenceDenormalizer
+    implements ChainableFixtureDenormalizerInterface, FlagParserAwareInterface
+{
+    use IsAServiceTrait;
+
+    /** @private */
+    const REGEX = '/.+\{(?<expression>@(?<name>([A-Za-z0-9-_]+))(?<flag>(\*+))?)\}/';
+
+    /**
+     * @var FlagParserInterface|null
+     */
+    private $flagParser;
+
+    /**
+     * @var SpecificationsDenormalizerInterface
+     */
+    private $specsDenormalizer;
+
+    public function __construct(SpecificationsDenormalizerInterface $specsDenormalizer, FlagParserInterface $parser = null)
+    {
+        $this->specsDenormalizer = $specsDenormalizer;
+        $this->flagParser = $parser;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function withFlagParser(FlagParserInterface $parser): self
+    {
+        return new self($this->specsDenormalizer, $parser);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function canDenormalize(string $name, array &$matches = []): bool
+    {
+        return 1 === preg_match(self::REGEX, $name, $matches);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function denormalize(
+        FixtureBag $builtFixtures,
+        string $className,
+        string $fixtureId,
+        array $specs,
+        FlagBag $flags
+    ): FixtureBag {
+        $matches = [];
+        if (false === $this->canDenormalize($fixtureId, $matches)) {
+            throw LogicExceptionFactory::createForCannotDenormalizerForChainableFixtureBuilderDenormalizer(__METHOD__);
+        }
+
+        $referencedName = $matches['name'];
+        $allFlag = ($matches['flag'] ?? null) === '*';
+
+        $fixtureIds = $this->buildReferencedValues($builtFixtures, $referencedName, $allFlag);
+
+        $fixtureIdPrefix = $this->determineFixtureIdPrefix($fixtureId);
+
+        foreach ($fixtureIds as $referencedFixtureId => $valueForCurrent) {
+            $builtFixtures = $builtFixtures->with(
+                $this->buildFixture(
+                    $fixtureIdPrefix . $referencedFixtureId,
+                    $className,
+                    $specs,
+                    $flags,
+                    $valueForCurrent
+                )
+            );
+        }
+
+        return $builtFixtures;
+    }
+
+    /**
+     * @param FixtureBag $builtFixtures
```

Comment:
let's get rid of the phpdoc if it doesn't provide any additional value to the typehint :)